pkg/operator/sync_test.go

```diff
@@ -1,14 +1,152 @@
 package operator
 
 import (
+	"testing"
+
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/stretchr/testify/assert"
-	"testing"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes/fake"
+
+	mcfgv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
 )
 
-func TestIsCloudConfigRequired(t *testing.T) {
-	testInfra := configv1.Infrastructure{}
-	testInfra.Status.Platform = "None"
-	required := isCloudConfigRequired(&testInfra)
-	assert.False(t, required)
+func TestSyncCloudConfig(t *testing.T) {
```

Comment:
Did we lose the platform = "none" as a case?

Reply:
I am happy to add a test case for platform "None". Is there something special about that platform that you would like to me test?

Reply:
Add a test case for None platform with no kube-cloud-config.